src/DotNetWorker.Core/Http/HttpResponseDataExtensions.cs

```diff
@@ -77,7 +78,23 @@ public static Task WriteStringAsync(this HttpResponseData response, string value
         /// <returns>A <see cref="ValueTask"/> that represents the asynchronous operation.</returns>
         public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T instance, CancellationToken cancellationToken = default)
         {
-            return WriteAsJsonAsync(response, instance, "application/json; charset=utf-8", cancellationToken);
+            return WriteAsJsonAsync(response, instance, "application/json; charset=utf-8", HttpStatusCode.OK, cancellationToken);
+        }
+        
+        /// <summary>
+        /// Write the specified value as JSON to the response body using the provided <see cref="ObjectSerializer"/>.
+        /// The response content-type will be set to <code>application/json; charset=utf-8</code> and the status code set to 200.
```

Comment:
nit: Looks like this needs to be updated, as this will set to the status provided as the parameter.